src/components/Search.js

```diff
@@ -1,31 +1,42 @@
 import { Button } from '../utils/StyledComponents/Button';
 import { SearchInput } from '../utils/StyledComponents/SearchBar';
 import { SearchFormStyled } from '../utils/StyledComponents/SearchForm';
+import { useQuery, useQueryUpdate } from '../Context/QueryContext';
+import { QueryProvider } from '../Context/QueryContext';
+
+const SearchForm = ({ searchTerm, getVideoResult }) => {
+
+  const term = useQuery();
+  const changeQuery = useQueryUpdate();
```

Comment:
If you decide to use 1 context
```suggestion

 const {query, changeQuery } = useQuery();
```